bench/measure_time.py

```diff
@@ -0,0 +1,42 @@
+import os, sys, time, numpy 
```

Comment:
This file should be formatted with 4 spaces to match our other Python tools

Reply:
It should also have this header:

# This file is part of the Luau programming language and is licensed under MIT License; see LICENSE.txt for details